web/geolocation/geolocation.css

```diff
@@ -0,0 +1,227 @@
+body {
+    font-family: 'Poppins';
+}
+
+.custom-bubble {
+    padding: 4px 16px;
+    max-width: fit-content;
+    max-height: fit-content;
+    background-color: #EAF0F6;
+    border-radius: 50px;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 24px;
+    color: #33687B;
+}
+
+.new-location-padding {
+    padding: 4px 16px;
+}
+
+.input-group-append > button {
+    margin-left: 20px;
+}
+
+.btn-create-new {
+    background: #33687B;
+    border-radius: 8px;
+    padding: 8px 16px;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-size: 16px;
+    line-height: 24px;
+    color: #FFFFFF;
+}
+
+.btn-create-new:hover {
+    background: #155c76;
+    color: #FFFFFF;
+}
+
+.btn-invite {
+    background: #FFFFFF;
+    border: 1px solid #33687B;
+    border-radius: 8px;
+    padding: 8px 16px;
+    font-family: 'Poppins';
+    font-style: normal;
+    font-size: 16px;
+    line-height: 24px;
+    color: #33687B;
+}
+
+.form-switch {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: flex-start;
+    justify-content: space-between !important;
+    padding-left: 0px;
+}
+.form-check-label {
+    margin-right: 50px !important;
+}
+
+.form-check-input {
+    margin-right: 80px;
+}
+
+.form-check-input:checked {
+    background-color: #33687B;
+    border-color: #33687B;
+}
+
+.user-details-label {
+    font-size: 12px;
+    line-height: 18px;
+    letter-spacing: 0.01em;
+    color: #606060;
+}
+
+.user-details-button {
+    border: 1px solid #33687B;
+    color: #33687B;
+    font-size: 16px;
+    line-height: 24px;
+}
+
+#map-canvas-div {
+    height: 400px;
+    width: 100%;
+    display: flex;
+}
+
+#geo-channel-id {
+    font-family: 'Poppins';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #000000;
+    padding-right: 16px;
+}
+
+@media only screen and (max-width: 600px) {
+    #map-canvas-div {height: 191px;}
+    .form-check-input {margin-right: 150px;}
+}
+@media only screen and (min-width: 600px) {
+    #map-canvas-div {height: 394px;}
+    .form-check-input {margin-right: 80px;}
+}
+
+@media only screen and (min-width: 992px) {
+    #map-canvas-div {height: 524px;}
+    .form-check-input {margin-right: 80px;}
+}
+
+/* Button */
```

Comment:
Again, I imagine a lot of these button styles will live in a shared file but for now, this is fine.